data/human/culture conversations.txt

```diff
@@ -622,3 +622,72 @@ mission "Sol-Bound Shuttle"
 			label end
 			`	You say your goodbyes as you board your ships. Although you are both captains, the trajectory of your lives have taken very different courses.`
 				decline
+
+# Note about Earth Day and Earth Hour:
+# Earth Day (in 2023) is a day of awareness about climate change and occurs every 22nd of April. Earth Hour is an hour
+# of blackout around cities worldwide, also to bring awareness to climate change, and it's observed on the last Saturday
+# of March.
+
+# Since Earth Day has become a quasi-'national day' for Earthlings in Endless Sky, it would be interesting to have Earth
+# Hour be conflated with Earth Day as well, to demonstrate how people in a thousand years' time observe what we observe
+# today and how its meaning has changed.
+mission "Earth Day Blackout"
+	minor
+	landing
+	source Earth
+	to offer
+		day = 22
+		month = 5
+	on offer
+		conversation
+			`As you follow the tracking beacons towards Earth's central spaceport, you realize there's something very off; what was the bustling spaceport city is now completely still, and even though it is near midnight, the supertall buildings and skyscrapers are curiously dark; not a light is on for kilometers.`
+			`	However, the spaceport seems to still be operating, because the standard, automated procedures of signing landing documents, refuelling and recalibrating, and checking your ship's spaceworthiness were met and completed as you landed. But when you enter the spaceport terminal, there are wax candles spaced out to bring some light into the now dark corridors. It's a strange sight for the most populated planet in human space.`
+			choice
+				`	(Find someone and ask.)`
+				`	(The blackout will probably pass.)`
+					goto pass
+			branch attendant tourist
+				random > 50
+
+			label attendant
+			`	You approach an information desk in the terminal's atrium that, given the situation, doesn't seem to be as busy as expected. After a moment you get to the attendant behind the counter.`
+			choice
+				`	"What's with the blackout?"`
+				`	"When is the blackout going to be fixed?"`
+			`	The attendant chuckles to himself. "I should have expected that question from your accent. Today's Earth Day, when all the cities of Earth - and I mean all of them - participate in a total blackout. Even the Parliament complex was out today. It makes us think about our origins; that we once lived on the soil without electricity, all confined to just this one planet here, divided and uncertain. Of course, people like me still have to keep working, but for other people it's a day off to enjoy what little wildlife and earth us Earthlings have left, ironically. Do you have any other questions?`
```

Comment:
```suggestion
			`	The attendant chuckles to himself. "I should have expected that question from your accent. Today's Earth Day, when all the cities of Earth - and I mean all of them - participate in a total blackout. Even the Parliament complex was out today. It makes us think about our origins; that we once lived on the soil without electricity, all confined to just this one planet here, divided and uncertain. Of course, people like me still have to keep working, but for other people it's a day off to enjoy what little wildlife and Earth us Earthlings have left, ironically. Do you have any other questions?`
```

Reply:
This could go either way, but I left it as is when I looked it over because I figured it was trying to be funny with "eart"h as in "dirt" as a nature reference than a literal concern of space on the planet Earth.

But if that joking usage isn't coming through reliably for people, probably is best to change it as suggested.